### DISCUSSION TITLE
problem in loop modeling

### DISCUSSION THREAD

**Discussion by croshong** [RosettaCommons/rosetta#77] [Rosetta]
Hi I'm now trying to insert loop in PDB.
my blueprint file is below
.
.
80 N .
81 S .
82 G L PIKAA G
0 X L PIKAA R
0 X L PIKAA A
83 V L PIKAA V
84 T .
85 Y .
.
.
as shown above I try to insert RA between G82 and V83 but in the result PDB the residue was 
SVVVVTY instead of SGRAVTY. what could be the problem for this error?
Attached files are input PDB and blueprint file , flags file for loop remodel, log file

[4k24U_0001.pdb.txt](https://github.com/user-attachments/files/15985737/4k24U_0001.pdb.txt)
[4k24U.remodel.txt](https://github.com/user-attachments/files/15985738/4k24U.remodel.txt)
[4k24U.log](https://github.com/user-attachments/files/15985739/4k24U.log)
[4k24U.remodel.txt](https://github.com/user-attachments/files/15985741/4k24U.remodel.txt)

**Comment by roccomoretti** [comment]
Valine is used as a "middle ground" residue in remodel during the early stages. I'm not sure about details here, but I would suspect the reason you're seeing V end up in the output PDB is that whatever particular remodel protocol you're using is only invoking some of those earlier backbone-focused stages, and you're not doing any later sidechain-involved stages.

This is borne out by the log, which is talking about "centroid" near the end, and is printing information about a centroid scoring function. My guess is that your output PDB may have a bunch of "CEN" atoms, rather than actual sidechain atoms.

There should be commands/options which will allow you to take your remodel blueprint file and use it to control a fullatom stage. That should not only fill out the sidechain atoms, but it will also change the residue identities to their proper settings.

**Comment by roccomoretti** [comment]
Marking this as "closed", as I hope your question has been answered. If that assessment is incorrect, feel free to re-open or open up a new discussion with more details.
